content/participants/ttjc@saiteja-bma/index.md

```diff
@@ -0,0 +1,54 @@
+---
+title: Sai Teja Bheema
+type: "participant"
+name: "Sai Teja Bheema"
+github-link: "https://github.com/saiteja-bma"
+linkedin-link: "https://www.linkedin.com/in/saitejabma"
+looking-for: "job"
+description: "self-taught developer, Skills: | HTML | CSS | Javascript | learning React | MachineLearning | Python  "
+---
+
+Hi! I am Sai Teja, 2019 B.E Passout self-learning web development. Let me walk you through my intro, projects and blogs.
+
+### introduction
+
+I come from a different background, I did my B.E in Electrical and Electronics Engineering, I started programming in python learning DataScience and MachineLearning through deploying applications I started developing interest in web development and started learning it. Thanks to Tanay and few others, today I have some basic understanding of how web works and have built some projects.
+  
+
+## why are you taking this challenge?
+
+I need a job so that I explore and enhance my skill set and gain a solid experience by working with some highly creative people in the industry.
+
+## something interesting about you?
+
+I'm a very curious person i keep learning anything that interests me. I love football, read books and play video games.
+
+### help I need?
+
+It would be good if someone can guide me in frontend and database sections.
+
+### how to contact me?
+
+- [Twitter](https://twitter.com/B_ma634)
+- [LinkedIn](https://www.linkedin.com/in/saitejabma)
+
+### projects
+
+1. Weather & Top Destinations Website :
+
+I had built a small website, where in user can view current weather and top destinations of all the cities in the world. 
+
+Technologies used:
+ *HTML | CSS | JavaScript
```

Comment:
Hey! @saiteja-bma if you want to add bullet points here give a space after * like:
```
* HTML | CSS | JavaScript
```